docs/reference/method/MongoDBGridFSBucket-openDownloadStream.txt

```diff
@@ -36,7 +36,28 @@ Errors/Exceptions
 .. include:: /includes/extracts/error-gridfs-filenotfoundexception.rst
 .. include:: /includes/extracts/error-driver-runtimeexception.rst
 
-.. todo: add examples
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $bucket = (new MongoDB\Client)->test->selectGridFSBucket();
+
+   $stream = fopen('php://temp', 'w+b');
+   fwrite($stream, "foobar");
+   rewind($stream);
+
+   $id = $bucket->uploadFromStream('filename', $stream);
+
+   $bucket->openDownloadStream($id);
```

Comment:
Should this assign the return value so that we can call `stream_get_contents()` on _that_? As-is, it looks like you're just dumping the upload stream contents.

You can probably shares the same example as https://github.com/mongodb/mongo-php-library/pull/527/files#diff-84cbd79743e7c8f651c3c597a28b3cd0.